src/MongoDB.Analyzer/Core/AnalysisContext.cs

```diff
@@ -14,9 +14,10 @@
 
 namespace MongoDB.Analyzer.Core;
 
-internal record MongoAnalyzerContext(
+internal record AnalyzerContext(
```

Comment:
rename to `AnalysisContext` (same as the filename)

Reply:
Done